front/src/components/login.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+
+export default class Login extends Component {
+    constructor(props){
+        super(props);
+        this.state={
+            email: props.email? props.email : '',
+            password: props.password? props.password : '',
+            error: false
+        }
+        this.login = this.login.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(event) {
```

Comment:
Maybe something like that?
`    handleChange(event) {
        const {name, value} = event.target;
        this.setState({[name]: value});
    }`